src/cache.ts

```diff
@@ -14,7 +14,12 @@ export class Cache {
     pkg: Package,
     githubToken: string
   ): Promise<string | undefined> {
-    return await this.changelogCache.getUrlOrFind(pkg, githubToken)
+    try {
+      return await this.changelogCache.getUrlOrFind(pkg, githubToken)
+    } catch (e) {
+      core.warning(`cache: failed to get changelog url of ${pkg.name}; ${e}`)
+      return Promise.resolve(undefined)
```

Comment:
async functionなので↓のほうがいいと思います。

```suggestion
      return undefined
```